Gemfile

```diff
@@ -5,4 +5,7 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in cocina-models.gemspec
 gemspec
 
-gem 'byebug'
+group :development, :test do
```

Comment:
A development and test group doesn't make a ton of sense in the context of a gem since we never deploy this.